lerna.json

```diff
@@ -0,0 +1,23 @@
+{
+  "lerna": "2.7.1",
+  "version": "independent",
+  "changelog": {
+    "rootPath": ".",
+    "repo": "commercetools/commercetools-serverless-examples",
+    "labels": {
+      "Type: New Feature": ":rocket: New Feature",
+      "Type: Breaking Change": ":boom: Breaking Change",
+      "Type: Bug": ":bug: Bug Fix",
+      "Type: Enhancement": ":nail_care: Enhancement",
+      "Type: Documentation": ":memo: Documentation",
+      "Type: Maintenance": ":house: Maintenance"
+    },
+    "cacheDir": ".changelog",
+    "ignoreCommitters": ["renovate", "renovate-bot"]
+  },
+  "command": {
+    "publish": {
+      "message": "chore(release): releasing package"
+    }
+  }
+}
```

Comment:
lerna is a tool to manage monorepos. It looks through everything in `packages` for a `package.json` and installs things and does a bunch more.